pelican/server.py

```diff
@@ -23,16 +24,17 @@ def do_GET(self):
         for suffix in SUFFIXES:
             if not hasattr(self,'original_path'):
                 self.original_path = self.path
-            self.path = self.original_path + suffix
-            path = self.translate_path(self.path)
-            if os.path.exists(path):
-                srvmod.SimpleHTTPRequestHandler.do_GET(self)
-                logging.info("Found: %s" % self.path)
-                found = True
-                break
-            logging.info("Tried to find file %s, but it doesn't exist. ", self.path)
+            for strip_pattern in RSTRIP_PATTERNS:
+                self.path = self.original_path.rstrip(strip_pattern) + suffix
+                path = self.translate_path(self.path)
+                if os.path.exists(path):
+                    srvmod.SimpleHTTPRequestHandler.do_GET(self)
+                    logging.info("Found: %s" % self.path)
+                    found = True
+                    break
```

Comment:
Does this break from the inner-most loop only? If a path is found with the first suffix, and we `break`, will we still try out all the other suffixes?